docs/contributing/developing.rst

```diff
@@ -116,6 +116,35 @@ If your Elasticsearch instance is located somewhere else, you can set the
 
     $ ELASTICSEARCH_URL=http://my-elasticsearch-instance:9200 python runtests.py --elasticsearch
 
+**Browser and device support**
+
+Wagtail is meant to be used on a wide variety of devices and browsers. Supported browser / device versions include:
+
+=============  =============  =============
+Browser        Device/OS      Version(s)
+=============  =============  =============
+Mobile Safari  iOS Phone      Latest
+Mobile Safari  iOS Tablet     Latest
+Chrome         Android        Latest
+IE             Desktop        9, 10, 11
+Chrome         Desktop        Latest
+MS Edge        Desktop        Latest
+Firefox        Desktop        Latest
+Safari         macOS          Latest
+=============  =============  =============
+
+Those environments are the ones Wagtail *is tested* on. Our development standards ensure that the site is usable on other browsers **and will work on future browsers**. To test on IE, install virtual machines `made available by Microsoft <https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/>`_.
```

Comment:
Maybe for now we could say something like: "we aim for Wagtail to work on those environments"? 

For the future purpose I think we could benefit from integrating with SauceLabs or BrowserStack. But that would probably also require a sandbox environment to properly test the Wagtail admin.